backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/attribute/HUAttributeConstants.java

```diff
@@ -87,4 +77,7 @@ public static String sqlBestBeforeDate(final String huIdColumnName)
 		Check.assumeNotEmpty(huIdColumnName, "huIdColumnName is not empty");
 		return "\"de.metas.handlingunits\".huBestBeforeDate(" + huIdColumnName + ")";
 	}
+
+	public static final String SYSCONFIG_AutomaticallySetLotNumber = "de.metas.handlingunits.attributes.AutomaticallySetLotNumber";
+	public static final String SYSCONFIG_AutomaticallySetBestBeforeDate = "de.metas.handlingunits.attributes.AutomaticallySetBestBeforeDate";
```

Comment:
introduce IHUAttributesBL.isAutomaticallySetLotNumber() and isAutomaticallySetBestBeforeDate().
avoid having public sysconfig names.

Reply:
done